File: Program103.java
import java.lang.*;
import java.util.*;

class Pattern
{
     private int iRow, iCol;

     public Pattern(int iValue1, int iValue2)
     {
         this.iRow = iValue1;
         this.iCol = iValue2;
     }
     public void DisplayPattern()
     {
         int i = 0, j = 0;

         for(i = iRow; i >= 1; i--)
         {
             for(j = 1; j <= i; j++)
             {
                 System.out.print("*\t");
             }
             System.out.println("\n");
         }
     }
}

class Program103
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0, iValue2 = 0;
        
        System.out.println("Enter number of rows");
        iValue1 = sobj.nextInt();

        System.out.println("Enter number og columns");
        iValue2 = sobj.nextInt();

        Pattern obj = new Pattern(iValue1, iValue2);

        obj.DisplayPattern();

    }
}

File: Program103.c
#include<stdio.h>

void DisplayPattern(int iRow, int iCol)
{
    int i = 0, j = 0;
    
    for(i = iRow; i >= 1; i--)
    {
        for(j = 1; j <= i; j++)
        {
            printf("*\t");
        }
        printf("\n");
    }
}
int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("Enter number of rows\n");
    scanf("%d",&iValue1);

    printf("Enter number of columns\n");
    scanf("%d",&iValue2);

    DisplayPattern(iValue1, iValue2);

    return 0;
}